flow360/component/resource_base.py

```diff
@@ -142,6 +144,27 @@ def handle_none_str(cls, value):
         return value
 
 
+# Thread-local storage for suppression state
+_thread_local = threading.local()
+
+
+@contextmanager
+def skip_submit_reminder():
+    """
+    Manual override to suppress reminder in __init__
+    because we are using simualtion (or AKA V2) framework.
+    """
+    if not hasattr(_thread_local, "submission_reminder_suppressed"):
+        _thread_local.submission_reminder_suppressed = False
+
+    previous_state = _thread_local.submission_reminder_suppressed
+    _thread_local.submission_reminder_suppressed = True
+    try:
+        yield
+    finally:
+        _thread_local.submission_reminder_suppressed = previous_state
+
+
```

Comment:
maybe we can just remove the reminder instead of this? This python API is supposed to be used for project submissions so reminder is not needed at all

Reply:
Sure!

Reply:
Removed